src/app/address/controllers/address/prepopulate.js

```diff
@@ -1,5 +1,5 @@
 const BaseController = require("hmpo-form-wizard").Controller;
-const logger = require("hmpo-logger").get();
+const logger = require("hmpo-logger").get("di-ipv-cri-address-front");
```

Comment:
Shall we make this a constant somewhere?

Reply:
It's a bit tricky given how the project is JS not TS. When I try create a constant and export it, errors start spewing up and when using the export in the logger, it's also doing some weird behaviour. This might be my lack of JavaScript knowledge but it seems like it'll need some work to get that working. Happy to pair if you want this done or we can leave it?

Reply:
Update: Done it!